### ISSUE TITLE
Error: FTS Xapian: Optimize UID inexistant

### ISSUE THREAD

**Issue by Philippe23** [grosjo/fts-xapian#123]
I've started getting a few errors like these within the last month:

```
doveadm(XXX): Error: FTS Xapian: Optimize UID=5449 inexistant
doveadm(XXX): Error: FTS Xapian: Optimize UID=486212 inexistant
doveadm(XXX): Error: FTS Xapian: Optimize UID=486215 inexistant
```

The "XXX" user has always been the same so far.

What does it mean and what can I do about it?

**Comment by grosjo** [comment]
Kindly test latest git

**Comment by Philippe23** [comment]
I'm running latest now.  I'll report back when I see another error.

**Comment by symphorien** [comment]
I also get about 500 of such errors when running `doveadm fts optimize -A` as recommended by the readme. I'm on version 1.5.5
Upgrading to current master 438b4e474ee26fa4606c0b06f8eb69f1cd6e0f55 fixes this message.

**Comment by bsdice** [comment]
@grosjo Hi... could you please release a new version with this fix so Arch picks it up? Getting 2000 lines each night. ;-) Thanks!

**Comment by grosjo** [comment]
Just put verbose=0 in the conf file

fts_xapian = partial=2 full=20 verbose=0

**Comment by symphorien** [comment]
ah so it's just a harmless warning?

**Comment by bsdice** [comment]
Hrm... Arch has 1.5.5 from 2022-04-17, also I already had verbose=0 active but messages are still present. Merci bien!

**Comment by Philippe23** [comment]
> I'm running latest now. I'll report back when I see another error.

Just to report back in: since building against HEAD sometime near the end of April, I have not had these errors re-occur.

Cutting a 1.5.6 would get this fix to more people.  (nudge, nudge)

**Comment by nachtbraker** [comment]
@grosjo  I'm seeing the same problem on FreeBSD. I agree that a new release would fix this, otherwise the pkg for FreeBSD will not receive this fix.

**Comment by brittAnderson** [comment]
Despite having set verbose=0 in the conf and using version 1.5.5 I am seeing these errors as well. Using archlinux package for installation. Happy to provide additional details if useful. Would be happy just understanding what is causing these errors so I know whether I can safely ignore. Thank you.

**Comment by grosjo** [comment]
Ok, will issue new package

**Comment by grosjo** [comment]
Released 1.5.6
